README.md

```diff
@@ -39,12 +39,12 @@ steps:
 ```yml
 version: 2.1
 orbs:
-  matlab: mathworks/matlab@0.1.4
-  codecov: codecov/codecov@1.1.1
+  matlab: mathworks/matlab@0.4.0
+  codecov: codecov/codecov@1.2.0
```

Comment:
```suggestion
  matlab: mathworks/matlab@0
  codecov: codecov/codecov@1
```

I would just specify the major version on these so this document doesn't get out-of-date too often.

Reply:
Good idea.